flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/AggregateUtil.scala

```diff
@@ -1208,5 +1208,40 @@ object AggregateUtil {
   private def gcd(a: Long, b: Long): Long = {
     if (b == 0) a else gcd(b, a % b)
   }
-}
+  
+  /**
+   * Create an [[org.apache.flink.streaming.api.functions.ProcessFunction]] to evaluate final
+   * aggregate value over a window with processing time boundaries.
+   *
+   * @param namedAggregates List of calls to aggregate functions and their output field names
+   * @param inputType Input row type
+   * @param timeBoundary time limit of the window boundary expressed in milliseconds
+   * @param isPartitioned Flag to indicate whether the input is partitioned or not
+   * @return [[org.apache.flink.streaming.api.functions.ProcessFunction]]
+   */
+  private[flink] def createTimeBoundedProcessingOverProcessFunction(
+    namedAggregates: Seq[CalcitePair[AggregateCall, String]],
+    inputType: RelDataType,
+    timeBoundary: Long,
```

Comment:
@sunjincheng121  :) ...i feel it is more expressive to mention time as the window is related to time boundaries. Offset for me sounds more related to indexes...but i am not a native speaker though...